### PR TITLE
deployment: add docker compose down command in Makefile.

### PR DESCRIPTION
**Pull request by iFurySt**
- **Description:** Add a target to Makefile to stop all containers.

i'm not sure if using `stop -> docker compose down` is more appropriate compared to using `stop -> docker compose stop` and `down -> docker compose down`. please feel free to adjust according to community consensus and norms.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 dev:
 	@docker compose watch
+stop:
```
Comment (lusmoura) [inline Makefile:3]: Would you mind renaming the command to `down` instead of `stop`?

**Review by lusmoura — APPROVED** [review]
Thank you for your contribution! We appreciate it and hope to see more of your contributions in the future 😄

**Comment by tianjing-li** [comment]
@iFurySt you'll need to sign your commits, see: https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

Then run 

```
git reset $(git merge-base main $(git branch --show-current))
^it undos all your commits and adds it to you local changes
then
git commit -S -m "your message"
```

**Comment by iFurySt** [comment]
> @iFurySt you'll need to sign your commits

done.

**Comment by tianjing-li** [comment]
Merging this

**Pull request merged**